AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/Model/cryptographic-materials-cache.smithy

```diff
@@ -215,7 +215,8 @@ union CacheType {
   No: NoCache,
   SingleThreaded: SingleThreadedCache,
   MultiThreaded: MultiThreadedCache,
-  StormTracking: StormTrackingCache
+  StormTracking: StormTrackingCache,
+  Shared: CryptographicMaterialsCacheReference
```

Comment:
If we are going down this path, 
which I am not strongly opposed to, 
we should not call this member `Shared`.

`Initialized` or `InitializedCMC` make more sense to me.

Reply:
I think `InitializedCache` would make more sense. I don't know how much we broadcast the name CMC.